src/account_multisig.ts

```diff
@@ -157,7 +216,34 @@ export class AccountMultisig extends Account {
         return requestId
     }
 
-    async verifyRequestCode(securityCode: string) {
+    async getCodeDefault(): Promise<string> {
+        if (typeof window === 'undefined') {
+            throw new Error('There is no getCodeFallback for Node env. Please provide your own in AccountMultisig constructor options argument.');
+        }
+        const method = await this.get2faMethod();
+        if (!method || !method.kind) {
+            throw new Error('no active 2fa method found');
+        }
+        const code = window.prompt(`Enter security code sent to ${ method.kind.split('2fa-')[1] }`)
+        return code
+    }
+
+    async promptAndVerify() {
+        const method = await this.get2faMethod();
+        const securityCode = await this.getCode(method)
+        try {
+            const { success, res: result } = await this.verifyCode(securityCode);
+            if (!success || result === false) {
+                throw new Error('Request failed with error: ' + JSON.stringify(result));
```

Comment:
what kind of errors won't result in `this.verifyCode` throwing exception?

Reply:
If contract panics, RPC will throw in CH and set success to false with JSON stringified result.

Sometimes a contract will return boolean false. This will be in result.

Reply:
maybe I shouldn't include the false condition. Since some contracts might return false, but intention is that the request was processed just returned false.